RFCs/RFC-0: Lightweight RFC process.md

```diff
@@ -0,0 +1,69 @@
+# RFC-0: Lightweight RFC Process for riff
+
+**To be Reviewed By:** Nov 15th 2019
+
+**Authors:** Swapnil Bawaskar
+
+**Status:** **Active** | Accepted | Dropped | Superseded
+
+**Superseded by:** N/A
+
+**Related:** N/A
+
+
+## Problem
+As the scope of the project expands and the number of moving pieces increase, we need a process to gather feedback on the changes proposed to the architecture and interaction of various components in the system.
+
+This document will refer to that process as the “Request For Comments (RFC) process”.
+
+## Solution
+The proposed solution to address the problem described above is to have an individual author or a group of authors submit a proposal to the community in order to gather feedback and achieve consensus. The RFC follows the same format as used by this proposal.
+
+Much inspiration for this proposal has been drawn from [Apache Geode's RFC process](https://cwiki.apache.org/confluence/display/GEODE/Lightweight+RFC+Process), which inturn draws from Phil Calçado’s [Structured RFC Process](https://philcalcado.com/2018/11/19/a_structured_rfc_process.html).
+
+All RFCs are submitted via PRs which will be merged when approved.
+
+### Collaboration
+Comments and feedback should be provided on the PR and focus on the technical content.
```

Comment:
As opposed to what?

Reply:
Agreed, while I see why you put that sentence in, I feel this could be misinterpreted. Not all RFCs are about "technical content". And even when it is technical, it is not "only technical" (ie execution) but should also question other aspects beforehand (eg motivation)

Reply:
There was a sentence in the Geode RFC, about not to comment on grammar, language etc, that I had removed. 
All fixed now.